03_probability_intro/03_probability_intro.jl

```diff
@@ -0,0 +1,443 @@
+
+$P(\text{R and V}) = P(R)P(V).$
+
+Coloquially, this means that the probability of both of the events 'it will rain in Buenos Aires today' and 'the volcano will erupt today' happening is equal to the product of each of the probabilities we know separately. Calculating the probability of various events from happening at the same time is known as calculating the *joint probability* of the events. When the events are not independent from one another, unfortunately, things are not so simple. This leads us into another important concept, and really fundamental to the Bayesian interpretation of probability, which we will discuss later on. 
+When events are not independent, it is relevant to talk about *conditional probability*. Conditional probability of two events A and B is noted as
+
+$P(A|B)\text{ or }P(B|A)$
+
+In general, $P(A|B)$ and $P(B|A)$ are not equal, that's why I have written the two possibilities. The way to interpret this, for example $P(A|B)$, is: 'the probability of the event A happening, *given* that we know the event B occured'. The analogous interpretation for $P(B|A)$ would be 'the probability of event B happening, given that we know the event A occured'. Let's see in a simple example how this conditional probability arises from two not independent events. Assume we have, again, the event R 'it will rain today in Buenos Aires', and another event H, 'staying at home today'. Applying what we learned, we would first ask ourselves if these two events are independent or interdependent. If they were completely independent one from another, when trying to compute the conditional probabilities, for example $P(R|H)$, we will have
+
+$P(R|H) = P(R)$
+
+What this equation is telling us, is that the information that the event H happened, does not affect the probability of event R. That is the bare definition of independence! However, this only means the event R is independent from the event H. This does not automatically mean that the event H is endependent of R. In fact, if we pay a little more attention to how this events relate one another, we have our answer.
+The probability of raining today in Buenos Aires won't change by the fact I stay at home or not. I can't change the climate, at least in that direct way. So effectively, $P(R|H) = P(R)$ is correct. But what if we invert the events? In this case, if we think about it for a second we will notice that
+
+$P(H|R) \neq P(H)$
+
+Indeed, if we know that it is raining, the probability of staying at home should be affected! In my case, I would much prefer to stay at home, so that probability will be higher. 
+At this point, we would state a formula that relates conditional probability and conjoint probability. For two general events A and B, this is
```

Comment:
the word is used in the [Think Bayes](https://www.greenteapress.com/thinkbayes/thinkbayes.pdf) book, page 2 of the book or page 20 of the pdf. However, it can be changed to "joint" if you find it more adequate